card/lib/cardio.rb

```diff
@@ -11,27 +11,11 @@ module Cardio
     Card::Loader.load_mods if Card.count > 0
   end
 
-  class << self
-    def paths
-      @@paths
-    end
-
-    def config
-      @@config
-    end
+  mattr_reader :paths, :config, :root, :cache
 
-    def root
-      @@root
-    end
-
-    def gem_root
-      CARD_GEM_ROOT
-    end
-
-    def add_card_paths paths, config, root
-      @@config = config
-      @@paths = paths
-      @@root = root
+  class << self
+    def card_config *args
+      @@config, @@paths, @@root, @@cache = args
```

Comment:
Most of this merged automatically when I merged 1_15, so I'm a bit puzzled about what happened here.  Maybe you could summarize what you changed in merging card-gem and I'll make sure it is reflected here.